app/lib/common/widgets/html_editor/html_editor.dart

```diff
@@ -48,6 +48,23 @@ extension ActerEditorStateHelpers on EditorState {
   String intoMarkdown({AppFlowyEditorMarkdownCodec? codec}) {
     return (codec ?? defaultMarkdownCodec).encode(document);
   }
+
+  /// clear the editor text with selection
+  void clear() async {
```

Comment:
this is utility fn added as extension to ```textEditorState``` to clear document content. Because it's usage is widely needed so having it neatly separated is so nice!!